claim.md

```diff
@@ -1,85 +1,89 @@
+* You'll first make a claim about yourself in the form of a `Claim` object;
+* But a claim in itself has no value. To become valid in the eyes of <span class="label-role verifier">verifiers</span>, it needs to be attested by an entity that <span class="label-role verifier">verifiers</span> trust: an <span class="label-role attester">attester</span>. So you'll create a `RequestForAttestation` object from your `Claim` object, so that an <span class="label-role attester">attester</span> can attest it.
 
+We'll look into the attestation in the next steps - for now, let's just focus on your claim.
 
-> 💡 KILT is permissionless.   
-> Anyone/anything can make a claim about themselves.
+> 💡 KILT is an open system.
+> Anyone/anything can make a claim about themselves. But a claim only has value if the verifier trusts the attester.
 
-<!-- and a *RequestForAttestaion* object, which we will share with the other participants, so that they can attest it. -->
-  
-## Get your identity as a claimer  
-In the previous Identity step in this tutorial, you've generated two identities.    
-You'll need the first mnemonic you've created; it's referred to as `<CLAIMER'S MNEMONIC>` in the code snippet below.   
+## Create a file
 
-## Code: create file 
-
-Create a new file `2-claim.js`. 
-All the following code needs to go into this file.  
+Create a new file `2-claim.js`.
+All of the code for this step needs to go into this file.
 
 ## Code: create a claim
-We'll create a claim using the provided ctype and the claimer identity.  
-Paste the following in `2-claim.js` (make sure to replace the `<CLAIMER'S MNEMONIC>`).  
 
-```javascript 
+In the previous step, you've generated two mnemonics and identities.
+You'll now need the first mnemonic you've created; it's referred to as `<CLAIMER'S MNEMONIC>` in the code snippet below.
+
+We'll create a claim using the provided CTYPE and the claimer identity.  
+Paste the following in `2-claim.js`. Make sure to replace the `<CLAIMER'S MNEMONIC>`.
+
+```javascript
 const Kilt = require('@kiltprotocol/sdk-js')
+
 // import the claim type file we've created previously
 const ctype = require('./ctype.json')
 
 // `<CLAIMER'S MNEMONIC>` is for example "gold upset segment cake universe carry demand comfort dawn invite element capital"
-const mnemonic = `<CLAIMER'S MNEMONIC>` 
+const mnemonic = `<CLAIMER'S MNEMONIC>`
 const claimer = Kilt.Identity.buildFromMnemonic(mnemonic)
 
-const rawClaim = {
+const claimContents = {
   name: 'Alice',
-  age: 29,
+  age: 25,
 }
 
 const claim = Kilt.Claim.fromCTypeAndClaimContents(
   ctype,
-  rawClaim,
+  claimContents,
   claimer.address,
   null
 );
 ```
 
 ## Code: create the `RequestForAttestation` object  
 
-We'd like our claim to be attested by a trusted entity.  
-To do so, we'll build a `RequestForAttestation` object, and send it to an attester.   
+Now that you have your claim, you want to sign it and prepare it for the attester.
```

Comment:
I was confused here because I didn't run the code at this point and you have stated we have a claim. 

Could we say that 'once you have executed the code you will receive a claim'? 

Or maybe, even adding an extra step to show what a claim will look like?

Reply:
OK great point, thanks - I've tried to make this clearer, let me know if that's better!

Also: 
I do like the idea of actually displaying the claim object in the workshop. _However_ this would be higher maintenance for us: the workshop would need to be updated each time we add a new feature to claims. I think we're not ready yet for this, it might turn out more confuaing than helpful; but we _could_ certainly do that once our documentation process / checks are a bit more solid! (e.g. CI on workshop examples)

Reply:
Ok, I hope that we dont forget about it. 

But you are right, we are doing a lot at the moment to these changes and seems redundant to keep updating it every other time.